src/components/fieldText/fieldText.module.scss

```diff
@@ -31,6 +31,7 @@
 .icon-container-start {
   @extend .icon-container;
   margin-right: 4px;
+  cursor: pointer;
```

Comment:
I'm not sure such a behavior always expected, may be just for `collapsible` field? Could you please clarify with UX team?